app/controllers/photos_controller.rb

```diff
@@ -76,4 +81,11 @@ def set_photo
   def photo_params
     params.require(:photo).permit(:image, :comments_count, :likes_count, :caption, :owner_id)
   end
+
```

Comment:
this is the method being called with the before action that is now no longer necessary thanks to pundit authorize. so can remove or comment out is my suggestion

```
def ensure_user_is_authorized 
    if !PhotoPolicy.new(current_user, @photo).show?
      raise Pundit::NotAuthorizedError, "not allowed"
    end
  end
``